frontend/src/lib/api/waitinglist.ts

```diff
@@ -0,0 +1,84 @@
+import type { Registration } from './registration';
+
+const BACKEND_URL = process.env.NEXT_PUBLIC_BACKEND_URL ?? 'http://localhost:8080';
+
+const WaitinglistAPI = {
+    checkIfCanPromote: async (slug: string, idToken: string): Promise<{ bool: boolean }> => {
```

Comment:
```suggestion
    checkIfCanPromote: async (slug: string, idToken: string): Promise<boolean> => {
```
Er vel ganske unødvendig å returnere et objekt med et felt når typen bare er `boolean`? Man forstår returverdi av funksjonsnavnet.